flink-autoscaler-standalone/src/test/java/org/apache/flink/autoscaler/standalone/AutoscalerStateStoreFactoryTest.java

```diff
@@ -64,19 +63,20 @@ void testCreateJdbcStateStoreWithoutURL() {
     @Test
     void testCreateJdbcStateStore() throws Exception {
         final var jdbcUrl = "jdbc:derby:memory:test";
-        DriverManager.getConnection(String.format("%s;create=true", jdbcUrl)).close();
-
-        // Test for create JDBC State store.
+        // Test for create JDBC Event Handler.
```

Comment:
This test class is related to state store instead of event handler.

Please check the comment, `STATE_STORE_TYPE` and `testCreateJdbcEventHandler`.